app/models/manageiq/providers/vmware/infra_manager/refresh_parser.rb

```diff
@@ -408,6 +470,34 @@ def self.host_inv_to_lan_hashes(inv, switch_uids)
           result_uids[uid] = new_result
           switch[:lans] << new_result
         end
+
+        dvportgroup_inv.to_miq_a.each do |data|
+          spec = data['config']
+          next if spec.nil?
+
+          # Find the switch to which this lan is connected
+          switch = switch_uids[spec['distributedVirtualSwitch']]
```

Comment:
This isn't working for me, this is what I get for config
`"config"=>{"key"=>"dvportgroup-92"}`
and switch_uids looks like this
`switch_uids.keys
[:pnic_id, "vSwitch0", "vSwitch1", "dvs-90"]`

so I'm getting switch = nil here